crates/walrus-service/src/node/events/event_processor.rs

```diff
@@ -856,19 +862,24 @@ impl EventProcessor {
         let committee_store = DBMap::reopen(
             database,
             Some(constants::COMMITTEE_STORE),
-            &ReadWriteOptions::default(),
+            &default_rw_opts,
             false,
         )?;
+        let event_store_opts = ReadWriteOptions::default().set_ignore_range_deletions(true);
```

Comment:
Oh, interesting. Event store is the only cf that is in use and set ignore range deletions to true. (init_state is not used currently).